output.R

```diff
@@ -173,6 +198,30 @@ if (comp == TRUE) {
     outputdirs <- outputdir
   }
 
+  # ask for filename_prefix, if one of the modules that use it is selected
+  modules_using_filename_prefix <- c("compareScenarios")
+  if (!exists("filename_prefix")) {
+    if (any(modules_using_filename_prefix %in% output)) {
+      filename_prefix <- choose_filename_prefix(modules = modules_using_filename_prefix[modules_using_filename_prefix %in% output])
+    } else {
+      filename_prefix <- ""
+    }
+  } else {
+    # because readArgs cannot treat empty strings, `no prefix` is specified by `-`
+    if (filename_prefix == "-") {
```

Comment:
does it mean, the users always need to type at least "-"?

Reply:
I added this because you could not use the command line option `Rstart output.R filename_prefix=` to get the old result without prefix. But [my now-merged PR](https://github.com/pik-piam/lucode2/pull/62) in lucode2 fixed this, so we can delete it.

Reply:
perfect